src/main/java/sa/assertj/iterables/Contains.java

```diff
@@ -0,0 +1,31 @@
+package sa.assertj.iterables;
+
+import sa.assertj.Experiment;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class Contains extends Experiment {
+    static Experiment.DataProvider provider = (size, numOfSamples) -> {
+        Object[][] result = new Object[numOfSamples][];
+        for (int i = 0; i < numOfSamples; i++) {
+            ArrayList<Integer> list = new ArrayList<>();
+            Random rand = new Random();
+            int element = rand.nextInt();
+            list.add(element);
```

Comment:
Always gonna be on the same position, I think let's just do something like in the `AnyMatch`.